### DISCUSSION TITLE
Don't understand 'previously created output folder'

### DISCUSSION THREAD

**Discussion by jimnto** [ocrmypdf/OCRmyPDF#1333] [Q&A]
How do I create an output folder to receive the out put of batch processing:
"This command will run ocrmypdf on all files named *.pdf in the current directory and write them to the previously created output/ folder. It will not search subdirectories."
From the documentation at: https://ocrmypdf.readthedocs.io/en/latest/cookbook.html but I don't see the actual command
Is it:
ocrmypdf *.pdf \name_of_folder

**Comment by jbarlow83** [comment]
You have to create all of the required folders on your own. If the folder is not found, it will not be created for you. This is a safety check.

**Reply by jimnto** [reply]
I ultimately could not get the batch processing to work.  I think I couldn't call parallel from the CMD line.  Maybe the path was wrong.  I did it all manually - awesome software.
